imports/api/classes/Server.jsx

```diff
@@ -613,16 +615,46 @@ export default class Server {
     }
 
     smppReceive(from, to, message) {
-        //this confirms that a new sms was received
-
-        //TODO: 
-        //get owner of the did number (didOwner function)
-        //get number of parts the message body has to calculate total sms price (use getParts)
-        //check if account is billable, if not set result to insufficient funds
-        //get application related to the did number (didApp function)
-        //construct object to be sent to the msg_url of the application
-        //charge the account (use manager class when inserting record)
-        //send the constructed object to msg_url using processRequestUrl
+        let owner = this.didOwner(from);
```

Comment:
It's smpp receive, so use the to parameter when checking didOwner because it is the number that's receiving a message.

Reply:
okay